ismatec/driver.py

```diff
@@ -55,39 +57,36 @@ async def __aexit__(self, *args):
         """Provide exit to the context manager."""
         self.hw._stop_event.set()
 
-    ####################################################################
-    # Properties or setters/getters                                    #
-    # one per channel for the ones that have the channel kwarg.        #
-    ####################################################################
```

Comment:
I find this style of comment (which I did a lot of in my C days) to never get maintained. It ends up being noise to all involved.

My current view is docstrings are almost the only place for comments because they at least have hope of eyeballs through auto-documentation and `help()`.